### PR TITLE
section card updates

### PR DESCRIPTION
**Pull request by PavelLaptev**
- added `clickable` and non `clickable` styles
- added hover state if it's clickable
- added `labelFor` where it was missing and removed where the card is not clickable

### CODE REVIEW

File: gitbutler-ui/src/lib/components/SectionCard.svelte [hunk 1]
```diff
@@ -27,6 +28,8 @@
 	class:loading={background == 'loading'}
 	class:success={background == 'success'}
 	class:error={background == 'error'}
+	class:clickable={labelFor !== ''}
```
Comment (mtsgrd) [inline gitbutler-ui/src/lib/components/SectionCard.svelte:31]: 🙏

File: gitbutler-ui/src/routes/settings/+page.svelte [hunk 2]
```diff
@@ -275,7 +275,7 @@
 				</div>
 			</SectionCard>
 
-			<SectionCard labelFor="signingSetting" orientation="row">
+			<SectionCard labelFor="signingSetting" orientation="row" for="signingSetting">
```
Comment (mtsgrd) [inline gitbutler-ui/src/routes/settings/+page.svelte]: I think you want to change `for` -> `labelFor`, but on this point.. I wanted to make the param `for`, but it's a reserved keyword in js.

Comment (PavelLaptev) [reply gitbutler-ui/src/routes/settings/+page.svelte]: right! thank you

File: gitbutler-ui/src/lib/components/AnalyticsSettings.svelte [hunk 3]
```diff
@@ -39,21 +39,25 @@
 	</div>
 
 	<div class="analytics-settings__actions">
-		<SectionCard on:click={toggleErrorReporting} orientation="row">
+		<SectionCard labelFor="errorReportngToggle" on:click={toggleErrorReporting} orientation="row">
```
Comment (mtsgrd) [inline gitbutler-ui/src/lib/components/AnalyticsSettings.svelte:42]: ❤️

**Pull request merged**